crowbar_framework/config/application.rb

```diff
@@ -19,7 +19,7 @@
 
 module Crowbar
   class Application < Rails::Application
-    SERVER_PID = %x[ps ax | grep "rainbows master" | grep -v grep].split(" ")[0]
+    SERVER_PID = %x[ps ax | grep "puma master" | grep -v grep].split(" ")[0]
```

Comment:
Style/UnneededPercentX: Do not use `%x` unless the command string contains backquotes.